org.eclipse.gef.tests/src/org/eclipse/gef/test/RulerLayoutTests.java

```diff
@@ -57,9 +77,12 @@ public void testConstraint() {
 	/**
 	 * Use Case: Constraint must be of type int -> error
 	 */
-	@Test(expected = IllegalArgumentException.class)
+	@Test
 	public void testInvalidConstraint() {
```

Comment:
Do we now need a test where we are giving an Integer and check that no message was logged?

Reply:
I suppose that's easy enough to do... Done.